README.md

```diff
@@ -36,7 +36,7 @@ class ApplicationController < ActionController::Base
   before_filter :initialize_klaviyo
 
   def initialize_klaviyo
-    @klaviyo = Klaviyo::Client.new("YOUR_KLAVIYO_API_TOKEN")
+    @klaviyo = Klaviyo::Client.new("YOUR_KLAVIYO_API_TOKEN", "YOUR_KLAVIYO_PRIVATE_API_TOKEN")
```

Comment:
could we use `'` instead of `"`

Reply:
Is there a style guide for this project? It seems like StandardRB and some other style guides I have viewed recently prefer double quotes.

If there is a style guide setting up a GH action for rubocop (or standardRB) should handle this.